test/openstreetmap-polygons.js

```diff
@@ -30,7 +30,7 @@ describe('openstreetmap-polygons', function () {
     _extractor.start(function (err, results) {
       _error = err;
       _results  = results;
-      done();
+      setTimeout(done, 30);
```

Comment:
that's the time it takes to finish writing the error file. if i don't have the timeout the test looking for that file fails. :(